### PR TITLE
HELPS-1878 Add files related to the User aptitude

### PR DESCRIPTION
**Pull request by BrettMugglinOlive**
Add the User aptitude to the vscode extension and to npx

### CODE REVIEW

**Comment by BrettMugglinOlive** [comment]
It looks like my auto formatter is changing some of the stuff like single vs double quotes around the different files :/

**Comment by JosefBud-olive** [comment]
Could you undo those changes to the quotation marks? Otherwise we're just going to go back and forth with changing that. We should just add eslint or prettier to create-loop, but can do that in a follow-up ticket if you feel that's out of scope 😄 

In the LDK repo we prefer single quotes and I feel like it would be good to have a uniform stance between the codebases, what do you think?

**Comment by BrettMugglinOlive** [comment]
> 

So, it's what my editor is doing by default whenever I save (so it's not a conscious choice). But I think you're right that it should be the single quotes.

File: vscode-extension/src/createLoopForm.html [hunk 1]
```diff
@@ -78,12 +78,17 @@ <h2>Create Loop</h2>
...
-        <button class="pas" onclick="openFolderBrowser()" type="button">Browse</button>
+        <button class="pas" onclick="openFolderBrowser()" type="button">
+          Browse
+        </button>
       </div>
       <div class="pam">
-        <label for="projectName"><span class="required">*</span>Project Name</label><br />
+        <label for="projectName"
+          ><span class="required">*</span>Project Name</label
+        ><br />
```
Comment (JosefBud-olive) [inline vscode-extension/src/createLoopForm.html:91]: I won't hold up the PR over this but some of these HTML tags got messed up like
```html
  </label
>
```

Comment (BrettMugglinOlive) [reply vscode-extension/src/createLoopForm.html]: This should have been addressed now. Though, it looks like some of those may have already been there

**Review by JosefBud-olive — APPROVED** [review]

**Pull request merged**